axelrod/strategies/axelrod_second.py

```diff
@@ -812,11 +813,13 @@ def strategy(self, opponent: Player) -> Action:
         if opponent.history[0] == D:
             number_defects -= 1
 
-        if number_defects in [4, 7, 9]: return D
+        if number_defects in [4, 7, 9]:
```

Comment:
Hey @gaffney2010, it looks like you've copied this manually from https://github.com/Axelrod-Python/Axelrod/pull/1143?

No big deal at all, I suspect you've done this to try and avoid merge conflicts going forward but for info, that's not considered best practice in terms of etiquette. A better way to do this would be to "go get" the actual git commits that make this change, there are various ways to do this (for example a command called `git cherry-pick`. One of the main reasons for this is that it ends up causing problems on other people's branches (as well as potentially erasing their contributions in terms of git commit logs).

Not a big problem at all, and not suggesting you change anything (although once #1143 goes in you might end up with a merge conflict here), just letting you know :+1: :)

Reply:
@gaffney2010 now that #1143 has gone in there are a couple of merge conflicts. Let me know if you need a hand :+1:

Reply:
Okay.  I can see why it would be better to go get the changes directly from the commit.  How can you tell that I did it manually.  Did I miss something or does it show up differently?

Reply:
No it shows up the same but the commit that made the code on #1143 isn't in the history here so the conclusion is that you did it manually.

Not a big deal :)